### PR TITLE
Update installation README

### PR DESCRIPTION
**Pull request by R-Lawton**
## Additional Information
The following PR is to verify that the changes made to update the README of the installation repo are accurate and relevant.
## Verification Steps
There are no verification steps required for this PR. All that is needed is to ensure the README makes sense and is relevant.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -16,7 +16,7 @@ toc::[]
 
 = Overview
 
-The purpose of this repository is to provide a set of Ansible playbooks that can be used to install a range of Red Hat middleware products on Openshift.
+The purpose of this repository is to provide a set of Ansible playbooks that can be used to install a range of Red Hat middleware products on an Openshift cluster.
```
Comment (ciaranRoche) [inline README.adoc]: Pretty sure we only support running the playbook either on the PDS bastion or via tower so maybe we can re-word this to something like 
"The purpose of this repository is to provide a set of Ansible playbooks that can be used to install a range of Red Hat middleware products via Ansible Tower or RHPDS"

File: README.adoc [hunk 2]
```diff
@@ -50,16 +50,36 @@ These products include:
 
 == Installation Steps
 
-The following section demonstrates how to install each of the products listed above on an existing Openshift cluster.
+The following will provide information on how to install Integreatly on a openshift cluster via a bastion host.
```
Comment (ciaranRoche) [inline README.adoc]: We can also reword this to say RHPDS. Might actually be worth adding a prereq to say we need a running RHPDS cluster.

Comment (R-Lawton) [reply README.adoc]: Ok cool ill reword this and the above comment and and to the prereq

File: README.adoc [hunk 3]
```diff
@@ -50,16 +50,36 @@ These products include:
...
+__NOTE__: This hostname will be needed in a following step to allow connection to your cluster
+
+[source,shell]
+----
+oc get nodes
+----
+
 == Create the inventory hosts file
 
 . Create the host file based on the template(`../inventories/hosts.default`). Following the an example.
```
Comment (dimitraz) [inline README.adoc]: Suggestion:
```suggestion
Create the host file based on the template found in the inventories directory (`../inventories/hosts.default`):
```

File: README.adoc [hunk 4]
```diff
@@ -108,14 +133,32 @@ master.example.openshiftworkshop.com | SUCCESS => {
...
+|http://localhost
+|Authorization callback URL
+|http://localhost
+|===
++
+IMPORTANT: The callback URL is a placeholder for now and will be changed after the installation playbook is finished.
+
+. Click on `Register Application`
+. The values found in GitHub OAuth App, `Client ID` and `Client Secret`, will be needed in the next step to install Integreatly.
+image::https://user-images.githubusercontent.com/7708031/48856868-7141d680-edaf-11e8-836f-4d533f8ed402.png[GitHub OAuth App Fields]
```
Comment (dimitraz) [inline README.adoc]: This image doesn't seem to be rendering correctly for me

File: README.adoc [hunk 5]
```diff
@@ -50,75 +49,123 @@ These products include:
...
 
-Following an example of the expected output.
+Output:
 
 [source,shell]
-----
-$ ansible -m ping all
+
+```
+`ansible -m ping all
```
Comment (ciaranRoche) [inline README.adoc]: Could we remove this back tick.

File: README.adoc [hunk 6]
```diff
@@ -244,15 +263,14 @@ For example, if the master url is `https://master.example.openshiftworkshop.com/
 
 image::https://user-images.githubusercontent.com/7708031/48856455-528f1000-edae-11e8-8c1a-f0b37a1049ce.png[integr8ly WebApp]
```
Comment (ciaranRoche) [inline README.adoc]: Unrelated to your changes, if you have a running cluster could we update this image to latest version of the web app?

File: README.adoc [hunk 7]
```diff
@@ -244,15 +263,14 @@ For example, if the master url is `https://master.example.openshiftworkshop.com/
...
 
-TIP: The project https://github.com/integr8ly/tutorial-web-app[Webapp] is responsible for the Integraly interface. You can find the URL looking for the router created for this project. As the following example.
+TIP: The project https://github.com/integr8ly/tutorial-web-app[Webapp] is responsible for the Integraly interface. You can find the URL looking for the router created for this project.
```
Comment (dimitraz) [inline README.adoc]:
```suggestion
TIP: The project https://github.com/integr8ly/tutorial-web-app[Webapp] is responsible for the Integreatly interface. You can find the URL looking for the router created for this project. 
```

File: README.adoc [hunk 8]
```diff
@@ -147,39 +194,12 @@ By default Mobile Developer Services will not be installed. In order to install
...
-image::https://user-images.githubusercontent.com/7708031/48856868-7141d680-edaf-11e8-836f-4d533f8ed402.png[GitHub OAuth App Fields]
 
-=== Run the playbook
+#### 8. Run the playbook
 
 [source,shell]
 ----
-$ oc login https://<openshift-master-url> -u <user> -p <password>
-$ ansible-playbook -i inventories/hosts playbooks/install.yml -e github_client_id=<your_client-id> -e github_client_secret=<your_client_secret>
+ ansible-playbook -i inventories/hosts playbooks/install.yml -e github_client_id=<your_client-id> -e github_client_secret=<your_client_secret>
```
Comment (dimitraz) [inline README.adoc]: I would also point out the `-e eval_self_signed_certs=true` flag for self signed certs (maybe as a note below alongside the other note)

File: README.adoc [hunk 9]
```diff
@@ -192,9 +212,9 @@ $ ansible-playbook -i inventories/hosts playbooks/install.yml
 ----
 ====
 
-=== Add the generated Authorization callback URL to GitHub OAuth
+#### 9. Add the generated Authorization callback URL to GitHub OAuth
 
-Following and example of the output made at the end of the playbook with this URL.
+Replace the authorization callback URL previously conatining a temporary localhost value with the output of the installation seen below.
```
Comment (dimitraz) [inline README.adoc]:
```suggestion
Replace the authorization callback URL previously containing a temporary localhost value with the output of the installation seen below.
```

File: README.adoc [hunk 10]
```diff
@@ -50,75 +49,123 @@ These products include:
...
 == Installation Steps
 
-The following section demonstrates how to install each of the products listed above on an existing Openshift cluster.
+The following will provide information on how to install Integreatly on a openshift cluster via a bastion host.
+
 
-:numbered:
-== Clone installation GIT repository locally
+//:numbered:
+#### 1. Sign into your bastion host and become root of the host
```
Comment (dimitraz) [inline README.adoc]: Not sure if we still need to add these numbers

File: README.adoc [hunk 11]
```diff
@@ -50,75 +49,123 @@ These products include:
 
 == Installation Steps
 
-The following section demonstrates how to install each of the products listed above on an existing Openshift cluster.
+The following will provide information on how to install Integreatly on a openshift cluster via a bastion host.
```
Comment (dimitraz) [inline README.adoc]:
```suggestion
The following will provide information on how to install Integreatly on an Openshift cluster via a bastion host.
```

File: README.adoc [hunk 12]
```diff
@@ -16,8 +16,7 @@ toc::[]
 
 = Overview
 
-The purpose of this repository is to provide a set of Ansible playbooks that can be used to install a range of Red Hat middleware products on Openshift.
-
+The purpose of this repository is to provide a set of Ansible playbooks that can be used to install a range of Red Hat middleware products via Ansible Tower or RHPDS
```
Comment (dimitraz) [inline README.adoc]: I would add a `.` at the end of this sentence because it renders as `Ansible Tower or RHPDS These products include`

File: README.adoc [hunk 13]
```diff
@@ -16,8 +16,7 @@ toc::[]
...
 These products include:
 
 * Single Sign On
```
Comment (dimitraz) [inline README.adoc]: There are a few more services here we could add, I think this list is outdated

File: README.adoc [hunk 14]
```diff
@@ -50,75 +49,123 @@ These products include:
...
+
+#### 3. Obtain your openshift master hostname using the following command
+
+
+__NOTE__: This hostname will be needed in a following step to allow connection to your cluster
+
+[source,shell]
+
+```
+oc get nodes
```
Comment (dimitraz) [inline README.adoc]: Could do an `oc get nodes | grep master` to just print the master node

File: README.adoc [hunk 15]
```diff
@@ -50,75 +49,123 @@ These products include:
...
 [OSEv3:children]
 master
 
 [OSEv3:vars]
-ansible_user=evals
+ansible_user=ec2-user
 
 [master]
-master.evals.example.com
+127.0.0.1
```
Comment (dimitraz) [inline README.adoc]: We might want to leave this as `master.evals.example.com`, localhost might be misleading

File: README.adoc [hunk 16]
```diff
@@ -50,75 +49,123 @@ These products include:
...
 ----
-+
-NOTE: It is possible to add the variable `ansible_ssh_private_key_file` for the master host when the ssh connection requires a public key.(E.g `ansible_ssh_private_key_file=~/.ssh/ocp-workshop.pem`)
 
-== Check the connection with the OpenShift cluster
 
-Run the following command in order to check the connection with the OpenShift cluster from the root of the repository.
+#### 5. Check the connection with the OpenShift cluster
 
+Run the following command to check the connection with the OpenShift cluster.
```
Comment (dimitraz) [inline README.adoc]:
```suggestion
Run the following command to verify the connection to the OpenShift cluster.
```
Might be slightly more clear

File: README.adoc [hunk 17]
```diff
@@ -244,15 +263,14 @@ For example, if the master url is `https://master.example.openshiftworkshop.com/
 
 image::https://user-images.githubusercontent.com/7708031/48856455-528f1000-edae-11e8-8c1a-f0b37a1049ce.png[integr8ly WebApp]
 
-TIP: The project https://github.com/integr8ly/tutorial-web-app[Webapp] is responsible for the Integraly interface. You can find the URL looking for the router created for this project. As the following example.
+TIP: The project https://github.com/integr8ly/tutorial-web-app[Webapp] is responsible for the Integraly interface. You can find the URL looking for the router created for this project. 
 
 image::https://user-images.githubusercontent.com/7708031/48856461-5884f100-edae-11e8-92ca-ef4c93f8961f.png[integr8ly WebApp Router]
 
 Also, with the *evals* users created by the installer is possible to check the services in the OpenShift catalog.
```
Comment (dimitraz) [inline README.adoc]: This could probably be rephrased as well

File: README.adoc (hunk 6 above)
Comment (dimitraz) [reply README.adoc]: Yeah agreed, in general we should just avoid using screenshots if possible because they do tend to get outdated quite quickly

File: README.adoc (hunk 13 above)
Comment (R-Lawton) [reply README.adoc]: What are the names of the services and ill add it in

File: README.adoc (hunk 15 above)
Comment (R-Lawton) [reply README.adoc]: That was taken directly from the hosts.template. Thats what the user will see when they copy from hosts.template to hosts

File: README.adoc (hunk 8 above)
Comment (R-Lawton) [reply README.adoc]: ok didnt know about that flag will add it

File: README.adoc [hunk 18]
```diff
@@ -25,6 +25,17 @@ These products include:
 * Eclipse Che
 * Launcher
 * 3Scale
+* Fuse 
+* Rhsso
```
Comment (david-martin) [inline README.adoc]: Can remove Rhsso as it's the same as Single Sign On

File: README.adoc [hunk 19]
```diff
@@ -25,6 +25,17 @@ These products include:
...
+* Gitea
+* Apicurrito
```
Comment (david-martin) [inline README.adoc]: Apicurito

File: README.adoc [hunk 20]
```diff
@@ -25,6 +25,17 @@ These products include:
 * Eclipse Che
```
Comment (david-martin) [inline README.adoc]: Given that the list is made up of RH products, and non RH products/projects, can probably update the 'Overview' section above e.g. `can be used to install a range of Red Hat middleware products & other projects via Ansible Tower on OpenShift`

File: README.adoc (hunk 18 above)
Comment (R-Lawton) [reply README.adoc]: oh didnt know that, ill eventually know these acronyms haha
